src/components/pacbio/PacbioRunWellEdit.vue

```diff
@@ -355,6 +355,7 @@ export default {
     },
     async updatePoolBarcode(row, barcode) {
       const index = row.index
+      await this.$store.dispatch('traction/pacbio/runCreate/findPools', { barcode: barcode })
```

Comment:
If they manually type in a pool barcode in a well this method will be called every time they type a character. I do not think this will be noticeable from a user point of view but something to bear in mind.

Reply:
Following on from that I actually changed this behaviour [here](https://github.com/sanger/traction-ui/pull/1056/files#diff-f4defb31124e9cc0b7c5d31a5d29574579c504f6c046647a068e411a12bd8b11R190). Worth pulling develop back into branch and checking it still behaves ok.